src/Core/Style.js

```diff
@@ -160,6 +161,61 @@ function defineStyleProperty(style, category, name, value, defaultValue) {
     style[category][name] = value;
 }
 
+/**
+ * @class
+ * @classdesc StyleContext is a class to store all informations
```

Comment:
`StyleContext Stores a feature metadata needed for its style computation. (e.g. its coordinates, if it should fill, stroke or draw a point, etc.)`